### ISSUE TITLE
Add Support for Variable Rate Shading

### ISSUE THREAD

**Issue by Pinkyyyy** [Yellow-Dog-Man/Resonite-Issues#414]
### Is your feature request related to a problem? Please describe.

The game currently suffers from many performance issues that are currently being fixed up in various ways by users and developers alike. Getting more stable frames would make the game more enjoyable to play, having to deal with constant stuttering is not only annoying but even uncomfortable in VR.

### Describe the solution you'd like

Implementation or support for [VRS ( Variable Rate Shading ) and VRS2 by Nvidia](https://developer.nvidia.com/vrworks/graphics/variablerateshading)

Currently when I go into the Nvidia control panel it says that this feature "is not supported" by Resonite.exe.

Implementing this would potentially boost frames and make the text in the center, or where the user is looking if the headset has tracking, clearer while rendering bits outside of that area at lower resolution and lower quality. I think this would be a great thing to have!

### Describe alternatives you've considered

I am not aware of any alternatives, or mods that achieve this, if there are any please let me know, I'd love to try this feature out.

**Comment by DoubleStyx** [comment]
I don't believe Unity 2019 supports VRS. This will probably have to wait until the custom renderer is implemented.
